src/Symfony/Component/Translation/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 2.8.0
 -----
 
+ * deprecated FileDumper::format(), use FileDumper::formatCatalogue() instead.
```

Comment:
don't use it. Overwrite it (it is a protected method, so cannot be used directly)